handler/user_test.go

```diff
@@ -218,3 +218,26 @@ func TestDeleteUser(t *testing.T) {
 		}
 	})
 }
+
+func TestCreateUser(t *testing.T) {
```

Comment:
Test cases that fall under CreateUser should be in this format:
```
t.Run("test case", func(t *testing.T) {
...
}
```
You don't need to fully test every case right now, but it would be great if you could format it and do the tests in a separate PR. :)

Reply:
I was a little confused about this since the original tests for CreateUser were the only tests in db/user.go that weren't written in the form above. I'll change the tests to be written in that format. Thanks for your review.